aquarius/app/util.py

```diff
@@ -57,17 +57,17 @@ def get_timestamp():
 
 
 def get_curation_metadata(services):
-    return get_metadata_from_services(services)["attributes"]["curation"]
+    return get_metadata_from_services(services)["curation"]
 
 
 def get_main_metadata(services):
-    return get_metadata_from_services(services)["attributes"]["main"]
+    return get_metadata_from_services(services)["main"]
 
 
 def get_metadata_from_services(services):
     for service in services:
         if service["type"] == "metadata":
```

Comment:
do we need to check whether these exist? e.g. retrieve the keys with .get(key) instead of ['key']? Is there any way there could be ValueErrors here?

Reply:
good point, i've added some asserts